src/components/ValidateSubscriber/reducer/validateMaxSubscribersFormReducer.test.ts

```diff
@@ -0,0 +1,69 @@
+import {
+  INITIAL_STATE,
+  validateMaxSubscribersFormReducer,
+  VALIDATE_MAX_SUBSCRIBERS_FORM_ACTIONS,
+} from "./validateMaxSubscribersFormReducer";
+import { getMaxSubscribersData } from "../validate-max-subscribers-doubles";
+
+describe("validateMaxSubscribersFormReducer", () => {
+  it(`${VALIDATE_MAX_SUBSCRIBERS_FORM_ACTIONS.START_FETCH} action`, () => {
+    // Arrange
+    const action = { type: VALIDATE_MAX_SUBSCRIBERS_FORM_ACTIONS.START_FETCH };
+
+    // Act
+    const newState = validateMaxSubscribersFormReducer(INITIAL_STATE, action);
+
+    // Assert
+    expect(newState).toEqual({
+      ...INITIAL_STATE,
+      loading: true,
+    });
+  });
+
+  it(`${VALIDATE_MAX_SUBSCRIBERS_FORM_ACTIONS.FINISH_FETCH} action`, () => {
+    // Arrange
+    const action = {
+      type: VALIDATE_MAX_SUBSCRIBERS_FORM_ACTIONS.FINISH_FETCH,
+      payload: getMaxSubscribersData(),
+    };
+
+    // Act
+    const newState = validateMaxSubscribersFormReducer(INITIAL_STATE, action);
+
+    // Assert
+    expect(newState).toEqual({
+      ...INITIAL_STATE,
+      validationFormData: getMaxSubscribersData(),
```

Comment:
Creo que sería mejor mockear los datos en el test o al menos limitar el uso del double a un solo punto:

```suggestion
    // Arrange
    const demoData = getMaxSubscribersData();
    const action = {
      type: VALIDATE_MAX_SUBSCRIBERS_FORM_ACTIONS.FINISH_FETCH,
      payload: demoData,
    };

    // Act
    const newState = validateMaxSubscribersFormReducer(INITIAL_STATE, action);

    // Assert
    expect(newState).toEqual({
      ...INITIAL_STATE,
      validationFormData: demoData,
```